charts/chirpstack/README.md.gotmpl

```diff
@@ -0,0 +1,45 @@
+[modeline]: # ( vim: set ft=markdown: )
+{{ template "chart.header" . }}
+{{ template "chart.deprecationWarning" . }}
+
+{{ template "chart.badgesSection" . }}
+
+{{ template "chart.description" . }}
+
+{{ template "chart.homepageLine" . }}
+
+{{ template "chart.maintainersSection" . }}
+
+{{ template "chart.sourcesSection" . }}
+
+{{ template "chart.requirementsSection" . }}
+
+# Teuto-Chirpstack Helm-Chart
+
+This is an opinionated helm chart that will install a ChirpStack LoRaWAN Network
+Server to your cluster.
+
+## Configuration
+
+For this to work, you will have to provide some configuration through your
+`values.yaml` (or set them through CLI).
+
+```yaml
+chirpstack:
+  # This secret is used for generating login and API tokens, make sure this
+  # is never exposed. Changing this secret will invalidate all login and API
+  # tokens. The following command can be used to generate a random secret:
+  #   openssl rand -base64 32
+  apiSecret: CHANGEME
```

Comment:
I would say we should by default generate a value for this so the user doesn't have to think about this and also doesn't have to add secrets to their git repo when using gitops